openslides/core/static/js/core/base.js

```diff
@@ -100,6 +101,13 @@ angular.module('OpenSlidesApp.core', [
             // Loads all projector data
             Projector.findAll();
 
+            // Loads all chat messages data and their user_ids
+            ChatMessage.findAll().then( function(chatmessages) {
```

Comment:
Please add a permission check here. If the user is not a manager, then he gets a 403 what is not a problem, but it saves one request that is only needed for managers